ynput/OpenPype
# Unreal: Fix Render Creator with new Level Sequence wrong sequence name

## Changelog Description
Fixed an issue with the Render creator when creating a new level sequence.

## Additional info
The Level Sequence was named with the subset name, but the Level Sequence needs to be named as the asset, to retrieve information about the specific shot (frame range especially for the Validator).

## Testing notes:
1. Create a new Render instance with the option to create a new Level Sequence, and selecting the right asset.
![image](https://github.com/ynput/OpenPype/assets/1087869/bf0e752b-b105-4ef4-b7fb-ab22dd351d1e)
2. Start a render for that sequence.
3. Try publishing the render. The publish should be successful with no complaints from the validators.
